src/olympia/devhub/templates/devhub/addons/submit/done.html

```diff
@@ -32,10 +38,11 @@
         _("Edit version {0}")|fe(uploaded_version.version) }}</a>
   </p>
 {% else %}
-  <h3>{{ _("Version Signed") }}</h3>
+
+  <h3>{{ _("%s Signed"|format(submit_page)|title) }}</h3>
   <p>
-    {{ _("You’re done! This version is signed and ready to for self-distribution. "
-         "You can download it by clicking the button below.") }}
+    {{ _("You’re done! This %s is signed and ready to for self-distribution. "
```

Comment:
Same here, also, given the fact that `submit_page` isn't translated (unless I'm blind) I feel like this isn't so nice for translators too. Can we maybe introduce a `submit_page_name` or something?